frontside/pubspec.lock

```diff
@@ -153,7 +153,7 @@ packages:
       name: test_api
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.4.2"
+    version: "0.3.0"
   typed_data:
     dependency: transitive
     description:
```

Comment:
I'm not sure what you did but we don't want to downgrade any dependencies. Can you revert this? :)

Reply:
https://dart.dev/guides/libraries/private-files

Reply:
According to the docs we should keep it:
> For application packages, we recommend that you commit the pubspec.lock file. Saving pubspec.lock ensures that everyone working on the app uses the exact same versions.

Source: https://dart.dev/guides/libraries/private-files#pubspeclock

We're developing an application, not a library :)